userguide/system.rst

```diff
@@ -2800,8 +2800,9 @@ option to :guilabel:`System`. Some fields are modified in
 :guilabel:`Network` so that the peer IP address, peer hostname, and
 virtual IP can be configured. An extra section is added to
 :guilabel:`IPMI` to allow configuring :ref:`IPMI` for each units.
-Modified fields use *1* or *2* identify the |ctrlrs-term|. These numbers
-correspond to the |ctrlr-term| labels on the %brand% chassis.
+Modified fields use *This Controller*, *1*, or *2* identify the
```

Comment:
is this missing a word ("to")?

```suggestion
Modified fields use *This Controller*, *1*, or *2* to identify the
```

Reply:
resolved in a different pr and merged into this branch